### ISSUE TITLE
Cannot find module error

### ISSUE THREAD

**Issue by joaobzrr** [jsonnull/electron-trpc#85]
Hi,

For some reason all the imports in the README fail.

```js
// All of the following give "Cannot find module..." error
import { createIPCHandler } from "electron-trpc/main";
import { exposeElectronTRPC } from "electron-trpc/main";
import { ipcLink } from "electron-trpc/renderer";
```

Importing from `/dist` or `/src` compiles, but fails at runtime with the same error.

I'm using version 0.2.1 of electron-trpc.

**Comment by jsonnull** [comment]
Hey, thanks for raising the issue.

What tooling is giving you the error? TypeScript? Eslint? A packager?

Please note that the `exposeElectronTRPC` import is from `electron-trpc/main`, as the current GitHub README shows.

**Comment by joaobzrr** [comment]
> What tooling is giving you the error? TypeScript? Eslint? A packager?

Not sure. I'm using the [electron-react-webpack-typescript-2022](https://github.com/codesbiome/electron-react-webpack-typescript-2022) template and I'm simply running `npm run start`. Guess it's Typescript?

> Please note that the exposeElectronTRPC import is from electron-trpc/main, as the current GitHub README shows.

Sorry, that was a typo. Fixed it now.

**Comment by joaobzrr** [comment]
Here's what `node_modules/electron-trpc/` looks like:

```
│   LICENSE
│   package.json
│
├───dist
│       main.cjs
│       main.d.ts
│       main.mjs
│       renderer.cjs
│       renderer.d.ts
│       renderer.mjs
│
└───src
    │   constants.ts
    │   types.ts
    │
    ├───main
    │   │   createIPCHandler.ts
    │   │   exposeElectronTRPC.ts
    │   │   index.ts
    │   │   resolveIPCResponse.ts
    │   │   utils.ts
    │   │   vite.config.ts
    │   │
    │   └───__tests__
    │           resolveIPCResponse.test.ts
    │
    └───renderer
            index.ts
            ipcLink.ts
            tsconfig.json
            utils.ts
            vite.config.ts
````

**Comment by jsonnull** [comment]
Can you try changing `moduleResolution` in `tsconfig.json` from `node` to `node16`?

**Comment by joaobzrr** [comment]
That fixed it. Thanks!

**Comment by jsonnull** [comment]
Thanks for bringing this to my attention.

I learned recently that there's a backwards compatible solution I can implement in this package that wouldn't require that `node16` change. There's almost no downsides for me.

I aim to make that change soon, that should prevent this issue from occurring again and you'll be able to go back to using `node` module resolution if it benefits you.

**Comment by jsonnull** [comment]
This is fixed in `electron-trpc@0.3.2` in case you want to continue using `node` module resolution in TS.
